runbotics/runbotics-orchestrator-ui/src/src-app/components/cron/components/CustomSelect/CustomSelect.tsx

```diff
@@ -109,6 +109,8 @@ const CustomSelect: FC<CustomSelectProps> = ({
             renderValue={renderTag}
             autoWidth={false}
             disabled={disabled}
+            required
+            error={!value.length}
```

Comment:
is there appropriate error message?

Reply:
user has to now what did he/she done wrong to make it right. When that's added feel free to merge

Reply:
Consulted this with Ada and agreed that there is no need for this in this particular case - it is handled by browser after submit that this field is required and as long as it it empty the borders are red. If that is truly necessary let me know.

Reply:
If there are red borders around empty inputs it's correct